.pnpmfile.cjs

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    hooks: {
+        readPackage(pkg) {
+            if (pkg.name === '@web3-storage/w3up-client') {
+                pkg.version = '13.1.1';
+            }
+            if (pkg.name === '@w3ui/react') {
+                pkg.version = '2.3.0';
+            }
+            return pkg;
+        },
+    },
+};
```

Comment:
Hmm, what is this? Is it not enough to pin in `package.json`?

Reply:
@alanshaw that was my attempt to solve a lib conflict I'm getting with pnpm in my local environment. It didn't work so I will remove this.